packages/sanity/playwright-ct/tests/utils/testHelpers.tsx

```diff
@@ -35,29 +35,47 @@ export function testHelpers({
   page: PlaywrightTestArgs['page']
   testInfo: TestInfo
 }) {
-  const platformName = getPlatformName(testInfo.project.name) || process.platform
```

Comment:
**suggestion:** given that `getPlatformName` is no longer being used, we can remove it entirely from `testHelpers.tsx`

Reply:
Removed it